modules/example-apache-httpd-configuration.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-ldap-identity-provider.adoc
+
+[id="example-apache-httpd-configuration_{context}"]
+= Example Apache HTTPD configuration for basic identity providers
+
+The basic identify provider (IDP) configuration in {product-title} 4 requires
+that the IDP server respond with JSON for success and failures. You can use CGI
+scripting in Apache HTTPD to accomplish this. This section provides examples.
```

Comment:
These are the files that the user must create on the Apache HTTPD webserver? It isn't clear, to me at least, what to do with these samples. We are assuming that the users will know what the parameters are and how to provide the values?

Reply:
@bostrt Can you please help provide some clarity here?

Reply:
The JSON response and CGI scripts are provided in the examples further below. Is there something specific you are looking for or perhaps this paragraph can be worded better?